packages/superset-ui-preset-chart-xy/src/Line/createTooltip.tsx

```diff
@@ -31,13 +33,24 @@ export default function createTooltip(encoder: Encoder, allSeries: Series[]) {
                 .filter(({ key }) => series[key])
                 .concat()
                 .sort((a, b) => series[b.key].y - series[a.key].y)
-                .map(({ key, color }) => ({
+                .map(({ key, color, strokeDasharray }) => ({
                   key,
-                  keyStyle: {
-                    color,
-                    fontWeight: series[key] === datum ? 600 : 200,
-                  },
-                  value: encoder.channels.y.formatValue(series[key].y),
+                  keyColumn: (
+                    <>
+                      <svg width="12" height="8" style={MARK_STYLE}>
```

Comment:
should this glyph not match the legend? that seems ideal. it's tricky with the dashed line, tho. In that case I wonder if we should update the legend to be lines instead of circles or squares. wdyt?

Reply:
Perhaps should update the legend to accept mark type and display line as well.

Reply:
Making the legend customizable seems like a good chunk of work itself. I will make it a follow-up PR.